src/vsr/replica.zig

```diff
@@ -4929,11 +5094,7 @@ pub fn ReplicaType(
             }
 
             if (self.status == .view_change and self.log_view < self.view) {
-                if (self.primary_index(self.view) != self.replica and
-                    self.start_view_change_quorum and !self.do_view_change_quorum)
-                {
-                    self.send_do_view_change();
-                }
+                if (!self.do_view_change_quorum) self.send_do_view_change();
```

Comment:
DVCs used to be from replica → primary, but now they are broadcast to all other replicas.
The primary uses them for the same purpose as before. Backups use them to trigger their own DVCs.